src/BinSkim.Rules/InitializeStackProtection.cs

```diff
@@ -57,6 +58,12 @@ protected override IEnumerable<string> FormatIds
 
         public override AnalysisApplicability CanAnalyze(BinaryAnalyzerContext context, out string reasonForNotAnalyzing)
         {
+            PE portableExecutable = context.PE;
+            AnalysisApplicability result = AnalysisApplicability.NotApplicableToSpecifiedTarget;
+
+            reasonForNotAnalyzing = MetadataConditions.ImageIsBootBinary;

```

Comment:
Can this check be moved into the common helper StackProtectionUtilities,CommonCanAnalyze?